2020/09/06 22:49:31 [error] [exception.CException] exception 'CException' with message 'คุณสมบัติ "Project.id" ยังไม่ถูกกำหนด' in C:\xampp\htdocs\pea_track\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\pea_track\protected\views\project\_formUpdate.php(539): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\pea_track\protected\views\project\update.php(12): CController->renderPartial('_formUpdate', Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#8 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): CController->renderPartial('update', Array, true)
#10 C:\xampp\htdocs\pea_track\protected\controllers\ProjectController.php(1814): CController->render('update', Array)
#11 [internal function]: ProjectController->actionUpdate('344')
#12 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(ProjectController), Array)
#13 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ProjectController), Object(ReflectionMethod), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('update')
#23 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('project/update')
#24 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#25 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/pea_track/project/update/344
HTTP_REFERER=http://localhost/pea_track/project/index
---
2020/09/06 22:50:43 [error] [exception.CException] exception 'CException' with message 'คุณสมบัติ "Project.id" ยังไม่ถูกกำหนด' in C:\xampp\htdocs\pea_track\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\pea_track\protected\views\project\_formUpdate.php(539): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\pea_track\protected\views\project\update.php(12): CController->renderPartial('_formUpdate', Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#8 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): CController->renderPartial('update', Array, true)
#10 C:\xampp\htdocs\pea_track\protected\controllers\ProjectController.php(1814): CController->render('update', Array)
#11 [internal function]: ProjectController->actionUpdate('344')
#12 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(ProjectController), Array)
#13 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ProjectController), Object(ReflectionMethod), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('update')
#23 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('project/update')
#24 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#25 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/pea_track/project/update/344
HTTP_REFERER=http://localhost/pea_track/project/index
---
2020/09/07 11:55:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'ระบบไม่สามารถดำเนินการ "cancel" ตามคำสั่งได้' in C:\xampp\htdocs\pea_track\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(270): CController->missingAction('cancel')
#1 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('cancel')
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#3 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/pea_track/paymentProjectContract/cancel/73?note=
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/07 11:55:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'ระบบไม่สามารถดำเนินการ "cancel" ตามคำสั่งได้' in C:\xampp\htdocs\pea_track\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(270): CController->missingAction('cancel')
#1 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('cancel')
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#3 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/pea_track/paymentProjectContract/cancel/73?note=vvvvv
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/07 12:58:14 [error] [exception.CException] exception 'CException' with message 'คุณสมบัติ "Project.proj_manager_name" ยังไม่ถูกกำหนด' in C:\xampp\htdocs\pea_track\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\db\ar\CActiveRecord.php(145): CComponent->__get('proj_manager_na...')
#1 C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\_form.php(169): CActiveRecord->__get('proj_manager_na...')
#2 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\update.php(11): CController->renderPartial('_form', Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#8 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): CController->renderPartial('update', Array, true)
#10 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(315): CController->render('update', Array)
#11 [internal function]: PaymentProjectContractController->actionUpdate('401')
#12 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#13 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('update')
#23 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#24 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#25 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/pea_track/paymentProjectContract/update/401
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/09 23:14:14 [error] [php] Undefined variable: type_condition (C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\_form.php:180)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): PaymentProjectContractController->renderFile()
#1 C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\create.php(12): PaymentProjectContractController->renderPartial()
#2 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): PaymentProjectContractController->renderInternal()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): PaymentProjectContractController->renderFile()
#5 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): PaymentProjectContractController->renderPartial()
#6 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(222): PaymentProjectContractController->render()
#7 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionCreate()
#8 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): PaymentProjectContractController->runAction()
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): PaymentProjectContractController->filterAccessControl()
#13 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): PaymentProjectContractController->runActionWithFilters()
#16 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): PaymentProjectContractController->run()
#17 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#19 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/paymentProjectContract/create
in C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\_form.php (180)
in C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\create.php (12)
in C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php (222)
2020/09/09 23:14:40 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\_form.php:193)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): PaymentProjectContractController->renderFile()
#1 C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\create.php(12): PaymentProjectContractController->renderPartial()
#2 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): PaymentProjectContractController->renderInternal()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): PaymentProjectContractController->renderFile()
#5 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): PaymentProjectContractController->renderPartial()
#6 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(222): PaymentProjectContractController->render()
#7 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionCreate()
#8 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): PaymentProjectContractController->runAction()
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): PaymentProjectContractController->filterAccessControl()
#13 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): PaymentProjectContractController->runActionWithFilters()
#16 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): PaymentProjectContractController->run()
#17 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#19 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/paymentProjectContract/create
in C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\_form.php (193)
in C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\create.php (12)
in C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php (222)
2020/09/10 21:33:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\pea_track\protected\controllers\ProjectController.php:1912
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\controllers\ProjectController.php(997): ProjectController->loadModel('1195')
#1 [internal function]: ProjectController->actionUpdate('1195')
#2 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(ProjectController), Array)
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ProjectController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('update')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('project/update')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/project/update/1195
---
2020/09/10 21:48:30 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''314' at line 1. The SQL statement executed was: SELECT sum(pc_cost) as sum_total,pc_code,pc_T_percent FROM project_contract WHERE pc_proj_id='314.
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (163)
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (542)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/10 21:48:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand ล้มเหลวในการดำเนินการเรียกลักษณะของ SQL: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''314' at line 1. The SQL statement executed was: SELECT sum(pc_cost) as sum_total,pc_code,pc_T_percent FROM project_contract WHERE pc_proj_id='314' in C:\xampp\htdocs\pea_track\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php(163): CDbCommand->queryAll()
#2 C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php(542): NotifyController::gnotify()
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): NotifyController->actionIndex()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('notify/index')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/notify/index?tab=5
HTTP_REFERER=http://localhost/pea_track/site/index
---
2020/09/10 21:49:00 [error] [php] Division by zero (C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php:178)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): NotifyController->runAction()
#2 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#4 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): NotifyController->filterAccessControl()
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): NotifyController->runActionWithFilters()
#8 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): NotifyController->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#11 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/notify/index?tab=5
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (178)
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (542)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/10 21:50:34 [error] [php] Division by zero (C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php:178)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): NotifyController->runAction()
#2 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#4 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): NotifyController->filterAccessControl()
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): NotifyController->runActionWithFilters()
#8 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): NotifyController->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#11 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/notify/index?tab=5
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (178)
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (542)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/10 21:50:50 [error] [php] Division by zero (C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php:178)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): NotifyController->runAction()
#2 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#4 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): NotifyController->filterAccessControl()
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): NotifyController->runActionWithFilters()
#8 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): NotifyController->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#11 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/notify/index?tab=5
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (178)
in C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php (542)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/10 22:03:37 [error] [php] Object of class Notify could not be converted to string (C:\xampp\htdocs\pea_track\protected\models\Notify.php:264)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\base\CComponent.php(617): call_user_func_array()
#1 C:\xampp\htdocs\pea_track\framework\zii\widgets\grid\CDataColumn.php(137): TbDataColumn->evaluateExpression()
#2 C:\xampp\htdocs\pea_track\framework\zii\widgets\grid\CGridColumn.php(151): TbDataColumn->renderDataCellContent()
#3 C:\xampp\htdocs\pea_track\framework\zii\widgets\grid\CGridView.php(616): TbDataColumn->renderDataCell()
#4 C:\xampp\htdocs\pea_track\framework\zii\widgets\grid\CGridView.php(572): TbGridView->renderTableRow()
#5 C:\xampp\htdocs\pea_track\framework\zii\widgets\grid\CGridView.php(482): TbGridView->renderTableBody()
#6 C:\xampp\htdocs\pea_track\framework\zii\widgets\CBaseListView.php(167): TbGridView->renderItems()
#7 unknown(0): TbGridView->renderSection()
#8 C:\xampp\htdocs\pea_track\framework\zii\widgets\CBaseListView.php(150): preg_replace_callback()
#9 C:\xampp\htdocs\pea_track\framework\zii\widgets\CBaseListView.php(135): TbGridView->renderContent()
#10 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(173): TbGridView->run()
#11 C:\xampp\htdocs\pea_track\protected\views\notify\index.php(659): NotifyController->widget()
#12 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require()
#13 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): NotifyController->renderInternal()
#14 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): NotifyController->renderFile()
#15 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): NotifyController->renderPartial()
#16 C:\xampp\htdocs\pea_track\protected\controllers\NotifyController.php(553): NotifyController->render()
#17 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): NotifyController->actionIndex()
#18 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams()
#19 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): NotifyController->runAction()
#20 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#21 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#22 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): NotifyController->filterAccessControl()
#23 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#24 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#25 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): NotifyController->runActionWithFilters()
#26 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): NotifyController->run()
#27 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#28 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#29 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/notify/index?tab=5
in C:\xampp\htdocs\pea_track\protected\models\Notify.php (264)
in C:\xampp\htdocs\pea_track\protected\views\notify\index.php (639)
in C:\xampp\htdocs\pea_track\protected\views\notify\index.php (659)
2020/09/10 22:29:50 [error] [php] include(SMTP.php): failed to open stream: No such file or directory (C:\xampp\htdocs\pea_track\framework\YiiBase.php:427)
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\mail.php(42): spl_autoload_call()
#1 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require()
#2 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): PaymentProjectContractController->renderInternal()
#3 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): PaymentProjectContractController->renderFile()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): PaymentProjectContractController->renderPartial()
#5 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(62): PaymentProjectContractController->render()
#6 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionMail()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): PaymentProjectContractController->runAction()
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): PaymentProjectContractController->filterAccessControl()
#12 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): PaymentProjectContractController->runActionWithFilters()
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): PaymentProjectContractController->run()
#16 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#17 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/paymentProjectContract/mail
in C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\mail.php (42)
in C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php (62)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/10 22:41:07 [error] [exception.phpmailerException] exception 'phpmailerException' with message 'Invalid address:  (addAnAddress to): boybe7oo@gmail.com,pathomphong.p@mwa.co.th' in C:\xampp\htdocs\pea_track\PHPMailer\class.phpmailer.php:929
Stack trace:
#0 [internal function]: PHPMailer->addAnAddress('to', 'boybe7oo@gmail....', 'kittika')
#1 C:\xampp\htdocs\pea_track\PHPMailer\class.phpmailer.php(900): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\pea_track\PHPMailer\class.phpmailer.php(819): PHPMailer->addOrEnqueueAnAddress('to', 'boybe7oo@gmail....', 'kittika')
#3 C:\xampp\htdocs\pea_track\protected\views\paymentProjectContract\mail.php(38): PHPMailer->addAddress('boybe7oo@gmail....', 'kittika')
#4 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\pea_track\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(782): CController->renderPartial('mail', Array, true)
#8 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(62): CController->render('mail', Array)
#9 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionMail()
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('mail')
#19 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#20 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/pea_track/paymentProjectContract/mail
---
2020/09/11 19:46:50 [error] [php] Undefined index: fiscalyear (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:221)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/genServiceWord
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (221)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/12 15:53:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'ระบบไม่สามารถดำเนินการ "export" ตามคำสั่งได้' in C:\xampp\htdocs\pea_track\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(270): CController->missingAction('export')
#1 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#3 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 15:58:22 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 [internal function]: PaymentProjectContractController->actionExport('310')
#4 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#5 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#16 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 15:58:22 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 [internal function]: PaymentProjectContractController->actionExport('310')
#4 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#5 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#16 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 15:58:27 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 [internal function]: PaymentProjectContractController->actionExport('310')
#4 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#5 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#16 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 15:58:32 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 [internal function]: PaymentProjectContractController->actionExport('310')
#4 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#5 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#16 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 15:59:14 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 [internal function]: PaymentProjectContractController->actionExport('310')
#4 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#5 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#16 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 15:59:15 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 [internal function]: PaymentProjectContractController->actionExport('310')
#4 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#5 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#16 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 15:59:20 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 [internal function]: PaymentProjectContractController->actionExport('310')
#4 C:\xampp\htdocs\pea_track\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PaymentProjectContractController), Array)
#5 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PaymentProjectContractController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#15 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#16 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 16:00:06 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/310
HTTP_REFERER=http://localhost/pea_track/paymentProjectContract/index
---
2020/09/12 16:00:10 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:00:38 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:00:40 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:00:43 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:01:30 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:01:32 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:01:34 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(277): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:02:55 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:02:56 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:03:50 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:03:52 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:04:26 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:04:28 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:04:33 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:05:04 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:06:04 [error] [exception.PHPExcel_Calculation_Exception] exception 'PHPExcel_Calculation_Exception' with message 'Sheet1!E18 -> internal error' in C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Cell.php:300
Stack trace:
#0 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5\Worksheet.php(460): PHPExcel_Cell->getCalculatedValue()
#1 C:\xampp\htdocs\pea_track\protected\extensions\phpexcel\vendor\PHPExcel\Writer\Excel5.php(187): PHPExcel_Writer_Excel5_Worksheet->close()
#2 C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php(281): PHPExcel_Writer_Excel5->save('php://output')
#3 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): PaymentProjectContractController->actionExport()
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('export')
#13 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('paymentProjectC...')
#14 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/pea_track/paymentProjectContract/export/
---
2020/09/12 16:23:49 [error] [php] Undefined variable: id (C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php:186)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(133): PaymentProjectContractController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\pea_track\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\pea_track\framework\web\filters\CInlineFilter.php(58): PaymentProjectContractController->filterAccessControl()
#4 C:\xampp\htdocs\pea_track\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\pea_track\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): PaymentProjectContractController->runActionWithFilters()
#7 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): PaymentProjectContractController->run()
#8 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/paymentProjectContract/export/
in C:\xampp\htdocs\pea_track\protected\controllers\PaymentProjectContractController.php (186)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/15 13:35:00 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5848)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2019&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5848)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/15 15:10:37 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5848)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2019&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5848)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/15 15:10:57 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5848)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2019&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5848)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/15 15:11:58 [error] [exception.CException] exception 'CException' with message 'คุณสมบัติ "Vendor.name" ยังไม่ถูกกำหนด' in C:\xampp\htdocs\pea_track\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\db\ar\CActiveRecord.php(145): CComponent->__get('name')
#1 C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php(5850): CActiveRecord->__get('name')
#2 C:\xampp\htdocs\pea_track\framework\web\actions\CInlineAction.php(49): ReportController->actionIncomeExcel()
#3 C:\xampp\htdocs\pea_track\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): CController->run('IncomeExcel')
#7 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController('report/IncomeEx...')
#8 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\pea_track\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2019&monthEnd=09&yearEnd=2020
HTTP_REFERER=http://localhost/pea_track/report/income
---
2020/09/15 15:52:38 [error] [php] Undefined index: cost (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5853)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2019&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5853)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/15 16:05:07 [error] [php] Undefined index: cost (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5853)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2019&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5853)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/16 19:02:41 [error] [php] Undefined variable: datail (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5867)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2019&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5867)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/16 22:46:34 [error] [php] Undefined variable: rows (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5915)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2012&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5915)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/16 23:07:19 [error] [php] Undefined index: pj_id (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5948)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2012&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5948)
in C:\xampp\htdocs\pea_track\index.php (13)
2020/09/16 23:09:00 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php:5939)
Stack trace:
#0 C:\xampp\htdocs\pea_track\framework\web\CController.php(286): ReportController->runAction()
#1 C:\xampp\htdocs\pea_track\framework\web\CController.php(265): ReportController->runActionWithFilters()
#2 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(282): ReportController->run()
#3 C:\xampp\htdocs\pea_track\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\pea_track\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\pea_track\index.php(13): CWebApplication->run()
REQUEST_URI=/pea_track/report/IncomeExcel?monthStart=09&yearStart=2012&monthEnd=09&yearEnd=2020
in C:\xampp\htdocs\pea_track\protected\controllers\ReportController.php (5939)
in C:\xampp\htdocs\pea_track\index.php (13)
